contracts/upgradeable_contracts/amb_erc20_to_native/HomeAMBErc20ToNative.sol

```diff
@@ -202,13 +202,14 @@ contract HomeAMBErc20ToNative is BasicAMBErc20ToNative, BlockRewardBridge, HomeF
     }
 
     /**
-    * @dev Allows to transfer any locked token on this contract that is not part of the bridge operations.
-    * Native tokens are not allowed to be claimed.
-    * @param _token address of the token.
+    * @dev Allows to transfer any locked tokens or native coins on this contract.
+    * @param _token address of the token, address(0) for native coins.
     * @param _to address that will receive the locked tokens on this contract.
     */
     function claimTokens(address _token, address _to) external onlyIfUpgradeabilityOwner {
-        require(_token != address(0));
+        // Since native coins are being minted by the blockReward contract and burned by sending them to the address(0),
+        // they are not locked at the contract during the normal operation.
+        // So, any erc20 token and native coins can be safely claimed.
```

Comment:
it is not clear. what will happen if native tokens are sent by the `selfdestruct` call? They will not be burnt. But this imbalance must be addressed by usage the `fixMediatorBalance` functionality rather than by tokens claiming. So, I would leave the code restricting claiming the native tokens (token address == 0x0)

Reply:
I thought it makes sense to have these both options available in such non-normal operation case: either to send tokens to the foreign side, or to claim them on the home side. In both cases, only the upgradeability owner can call these methods, and only he/she decides the receiver address, so I don't think there is a lack of some security here.

Reply:
Well, consider to apply Occam's razor principle here and leave only one way to handle such situation in order to draw an unambiguous direction to solve the issue with native tokens assigned without invocation of the fallback method.